.github/workflows/ion-java-performance-regression-detector.yml

```diff
@@ -21,31 +21,50 @@ jobs:
         with:
           java-version: 1.8
 
-      - name: Checkout ion-java repository from the current commit
+      - name: Checkout the ion-java from the existing commit.
         uses: actions/checkout@v2
         with:
           fetch-depth: 2
           submodules: recursive
-      - run: git checkout HEAD^
+          run: git checkout HEAD^
 
-      - name: Build ion-java from the current commit
+      - name: Build ion-java based on the existing commit
         run: mvn clean install
 
-      - name: Build ion-java-benchmark-cli based on the current ion-java
+      - name: Checkout ion-java-benchmark-cli
         uses: actions/checkout@v2
         with:
-          repository: amzn/ion-java-benchmark-cli
-          ref: master
-      - run: mvn clean install
+          repository: linlin-s/ion-java-benchmark-cli
+          ref: final
+
+      - name: Build ion-java-benchmark-cli based on the current ion-java
+        run: mvn clean install
 
       - name: Check the version of ion-java
         run: java -jar target/ion-java-benchmark-cli-0.0.1-SNAPSHOT-jar-with-dependencies.jar --version
 
       - name: Generate test Ion Data
-        run: java -jar target/ion-java-benchmark-cli-0.0.1-SNAPSHOT-jar-with-dependencies.jar generate -S 500 -T string -f ion_text testWorkflow.ion
+        run: |
+          mkdir -p testData
```

Comment:
> Quick questions since there is a `-p` option here. Is it possible that the directory (e.g., testData) has already existed so we used `-p` to avoid failures here?

Yes, because it's the start of the workflow and no existing directory, even there is the "-p" option can help to avoid failure.